### ISSUE TITLE
About error reporting during testing

### ISSUE THREAD

**Issue by zhuangyihao1997** [QMME/StableVQA#4]
Thank you for your work. However, when I tested according to the steps you provided, why when training the model, img_feat = img_f.reshape(n, d*img_f.size(1))
When testing img_feat = torch.flatten(self.avg_pool3d(img_f),1) there is an error in this line of code. Is there any difference between training and testing? I look forward to your reply. Thank you very much.

**Comment by QMME** [comment]
You are right about the error, as this line of code in testing should be deleted. It has been corrected by now.

**Comment by zhuangyihao1997** [comment]
> You are right about the error, as this line of code in testing should be deleted. It has been corrected by now.

OK，thank you for your reply. Thanks again for your open source
